### PR TITLE
Make counters climbable

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

cleans up the prototypes because of parenting issues.
makes them properly vault able

the justification is that being able to hide behind counters and not deal with people climbing over is really boring and unintuitive.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: You can now climb on counters.

### CODE REVIEW

**Comment by deltanedas** [comment]
*HoP chuckles*
HoP says: "I'm in danger"

File: Resources/Prototypes/Entities/Structures/Furniture/Tables/base_structuretables.yml [hunk 1]
```diff
@@ -35,3 +35,25 @@
...
   - type: Clickable
+
+- type: entity
+  id: CounterBase
+  parent: TableBase
+  name: counter
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/Tables/base_structuretables.yml:46]: TableBase is netsync false you spud

File: Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml [hunk 2]
```diff
@@ -492,20 +492,86 @@
...
+    sprite: Structures/Furniture/Tables/countermetal.rsi
+  - type: Icon
+    sprite: Structures/Furniture/Tables/countermetal.rsi
+    state: full
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: IconSmooth
+    key: state
+    base: state_
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml:552]: Dupe

File: Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml [hunk 3]
```diff
@@ -492,20 +492,86 @@
...
+    sprite: Structures/Furniture/Tables/counterwood.rsi
+  - type: Icon
+    sprite: Structures/Furniture/Tables/counterwood.rsi
+    state: full
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Wood
+  - type: IconSmooth
+    key: state
+    base: state_
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml:509]: Dupe

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Review by DrSmugleaf — APPROVED** [review]

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**